tests/posit/connect/api.py

```diff
@@ -2,10 +2,8 @@
 
```

Comment:
Are the changes in this file needed for what you're doing? They don't seem connected, but maybe I'm missing something

Reply:
I had added them in an earlier PR and Taylor had pushed back. I've come to agree that the type requirements should be relaxed.

So yes, unrelated to this PR, but useful.